src/GovUk.Education.ExploreEducationStatistics.Content.Model/Database/ContentDbContext.cs

```diff
@@ -391,813 +306,6 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
-                                        },
-                                        Label = "Unauthorised absence rate",
-                                        Colour = "#4763a5",
-                                        Symbol = ChartLineSymbol.circle,
-                                        LineStyle = ChartLineStyle.solid
-                                    },
-                                    new ChartLegendItem
-                                    {
-                                        DataSet = new ChartLegendItemDataSet
-                                        {
-                                            Indicator = Indicator(SubjectIds[SubjectName.AbsenceByCharacteristic], IndicatorName.Overall_absence_rate),
-                                            Filters = new List<Guid>
-                                            {
-                                                FItem(SubjectIds[SubjectName.AbsenceByCharacteristic], FilterItemName.Characteristic__Total),
-                                                FItem(SubjectIds[SubjectName.AbsenceByCharacteristic], FilterItemName.School_Type__Total)
-                                            },
-                                        },
-                                        Label = "Overall absence rate",
-                                        Colour = "#f5a450",
-                                        Symbol = ChartLineSymbol.cross,
-                                        LineStyle = ChartLineStyle.solid
-                                    },
-                                    new ChartLegendItem
-                                    {
-                                        DataSet = new ChartLegendItemDataSet
-                                        {
-                                            Indicator = Indicator(SubjectIds[SubjectName.AbsenceByCharacteristic], IndicatorName.Authorised_absence_rate),
-                                            Filters = new List<Guid>
-                                            {
-                                                FItem(SubjectIds[SubjectName.AbsenceByCharacteristic], FilterItemName.Characteristic__Total),
-                                                FItem(SubjectIds[SubjectName.AbsenceByCharacteristic], FilterItemName.School_Type__Total)
-                                            },
-                                        },
-                                        Label = "Authorised absence rate",
-                                        Colour = "#005ea5",
-                                        Symbol = ChartLineSymbol.diamond,
-                                        LineStyle = ChartLineStyle.solid
-
-                                    }
-                                }
-                            }
-                        }
-                    }
-                },
-                // absence generic data blocks used in content
-                new DataBlock
```

Comment:
Just letting you know the reason why we retained this seed data in this file here in case you didn't realise this as it's not very obvious.

We retained a bare minimum set of  Release content data for all the entity types being persisted as JSON e.g. Data block query, data block table config, data block chart config, methodology.

This was so that when creating an EF core migration we had the chance to spot if some of that JSON data was mutating in the generated migration and could then do something about it on a wider scale to migrate data in the wild if necessary e.g. by introducing a SQL update or a custom migration endpoint to transform the JSON for _all_ rows in the Prod database.

Without tracking any data used by these JSON fields here, when changing the models there was no indication in the migration that any data in the wild might need updating. Since the fields were of type String before and String afterwards, they remain as VARCHAR in the database and nothing appeared to have changed.

Updates probably aren't dangerous where fields are added or removed but might be where they are renamed or their type is altered.

Admittedly this was a poor way of testing for unsuspecting problematic changes as not everyone is going to attempt to generate a migration with dotnet tools after every change, it's not commented to explain the reason for it existing, and it's done nothing but confuse everyone who has looked at this and thought it's still a hangover from go-live!